File: Java/src/main/java/via/sep3/group2/rest/orderapi/model/Order.java
package via.sep3.group2.rest.orderapi.model;

public class Order {
    private Long id;
    private String description;
    private double amount;
    private boolean delivered;

    public Order(Long id, String description, double amount, boolean delivered) {
        this.id = id;
        this.description = description;
        this.amount = amount;
        this.delivered = delivered;
    }

    public Order() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getAmount() {
        return amount;
    }

    public void setAmount(double amount) {
        this.amount = amount;
    }

    public boolean isDelivered() {
        return delivered;
    }

    public void setDelivered(boolean delivered) {
        this.delivered = delivered;
    }
}

File: CSharp/ModelClasses/Order.cs
namespace ModelClasses;

public class Order
{
    public long id { get; set; }
    public string description { get; set; }
    public double amount { get; set; }
    public bool delivered { get; set; }

    public Order(long id, string description, double amount, bool delivered)
    {
        this.id = id;
        this.description = description;
        this.amount = amount;
        this.delivered = delivered;
    }

    public Order()
    {
    }
    
    
}